src/stories/dropdown.stories.js

```diff
@@ -0,0 +1,116 @@
+import React from "react"
+import Dropdown from "../components/dropdown/index"
+import { action } from "@storybook/addon-actions"
+import { object, number, boolean, text } from "@storybook/addon-knobs"
```

Comment:
I understood we wanted to go without knobs? did you find them to be valuable after all and we actually want them in?

Reply:
i think @Averethel said he sees values in knobs

Reply:
For this particular component I could not find another way to make it work, but I can try harder :blush: However there is no Knobs pannel created, everything is under Controls pannel.

Reply:
Actually, everything works perfectly fine without them 🤦 I was looking for something complicated when actually it was very simple ! Should I push the changes (stories without knobs) ?

Reply:
@ralucacril I did, but @Marine-Berthier explained that there's now a different way to play around with component. Let's roll without it for now and if we see we need them - we act.

Reply:
a ok. my bad. then @Marine-Berthier roll them without the knobs if is working

Reply:
I realised with this component that I didn't really understand the purpose of knobs. I thought that knobs could be replaced by args as soon as it is now possible to play with args the same way you can play with knobs. Actually it is still 2 different things : knobs could still be used and won't appear as args in the code. I might be wrong though